src/crud.js

```diff
@@ -46,7 +46,7 @@ export default class Todo {
       listItem.id = task.index.toString().concat('-task');
       const checkbox = document.createElement('input');
       checkbox.type = 'checkbox';
-      checkbox.checked = task.completed;
+      checkbox.id = task.index.toString().concat('-check');
       listItem.appendChild(checkbox);
```

Comment:
- [ ] Kindly create a module folder and import a javascript file containing methods related to the status updates (completed: true/false).

Reply:
Hi Kossi @Kossi-stack ,

Thank you for your input. However, I already have created a separate file, interaction.js in the src folder, and imported it into the index.js file. interaction.js is a class containing the methods of clearing the completed tasks and also the method to update the status of the task whether complete or not.

May you, therefore, clarify your instruction on exactly what I am expected to do?

Regards